uiowa/uiowa
# i3786 - Genetics - Adapt Thesis Defense content type from grad.uiowa.edu split

Closes #3786 


# How to test


1. `blt ds --site=grad.uiowa.edu` - make sure thesis_defense content still exists and the thesis_defense config split is turned on
2. `blt ds --site=genetics.grad.uiowa.edu` - make sure thesis_defense content can be added - doesn't error on save
